telicent_lib/mapper.py

```diff
@@ -168,6 +169,16 @@ def run(self):
                 self.__print_source_status__(self.source)
                 self.update_status(Status.TERMINATED)
                 self.aborted()
+            except RuntimeError as e:
```

Comment:
Something feels a little off about duplicating the same code. Let me come back to you on this bit when I've seen the whole file